contributed_definitions/NXms_atom_set.nxdl.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXms_atom_set" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <symbols>
+        <doc>
+             The symbols used in the schema to specify e.g. dimensions of arrays.
+        </doc>
+    </symbols>
+    <doc>
+         A base class to wrap details about a spatial configuration of atoms.
```

Comment:
the choice of the name ms_... shall be clear from the definition text

Reply:
done